code/modules/client/preference_setup/loadout/loadout_utility.dm

```diff
@@ -65,4 +65,14 @@
 	for(var/fanny in typesof(/obj/item/weapon/storage/belt/fannypack))
 		var/obj/item/weapon/storage/belt/fannypack/fanny_type = fanny
 		fannys[initial(fanny_type.name)] = fanny_type
-	gear_tweaks += new/datum/gear_tweak/path(sortAssoc(fannys))
\ No newline at end of file
+	gear_tweaks += new/datum/gear_tweak/path(sortAssoc(fannys))
+
+/datum/gear/utility/translator //riskiest PR I've ever made.
```

Comment:
This is not fit for the general loadout.
It has relatively high tech levels and gives a immediate advantage to the players in the game.
(Be it for the use in the science destructive analyzer or for its ability to understand most languages)

Reply:
it doesn't have high tech levels. it has origin_tech = list(TECH_DATA = 3, TECH_ENGINEERING = 3)

Reply:
A good point, but with the addition of universal translators the recorders should loose the ability to act as universal translators

Reply:
that would make them useless for investigations (their primary function)

Reply:
How so ? You can always ask another crew member to help you translate if you do not speak the language.
_Or the universal translator_

Reply:
Another crew member cannot produce a transcript that is very hard to forge.

Reply:
Also, furthermore, they only translate the basic languages that crew use, and not even anything with nonverbal or signlang. (And you can only use them when they're in your hand)

Reply:
Again, this does not leave a transcript. It impacts the performance of both CCIA and detectives.